taskzerooneproject/Program.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace taskzerooneproject
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.ForegroundColor = ConsoleColor.Green;
+            Console.WriteLine("Малинка");
+
+            Console.WriteLine();
+            
+            Console.ForegroundColor = ConsoleColor.Yellow;
+
+            Console.WriteLine("Что очень любит детвора");
+            Console.WriteLine("Срывать с колючего куста");
+            Console.WriteLine("И собирать в корзинку?");
+            Console.WriteLine("Конечно же — малинку!");
+
+            Console.ResetColor();
+
+
+            Console.ReadKey();
```

Comment:
Две пустых строки в коде подряд затрудняет чтение кода, а одна в нужном месте, когда код разделяется на смысловые блоки, облегчает.
```suggestion
            Console.ResetColor();

            Console.ReadKey();
```